test/modules/TestResultCoordinator/Program.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestResultCoordinator
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            // TODO: Add Coordinator logic
```

Comment:
I think you at least should add shutdownhandle.init call

Reply:
Done